arduinoClient/actions.ino

```diff
@@ -0,0 +1,108 @@
+enum ActionType {
+  CALL = 1,
+  TEXT = 2
+};
+
+struct Action {
+  ActionType type;
+  const char *phone_number;
```

Comment:
Minor comment: camelCase and underscore_separation are mixed. Doesn't have to be fixed, but would be nice.